src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -15036,6 +15036,38 @@ static void Main()
         }
 
         [Fact]
+        [WorkItem(14600, "https://github.com/dotnet/roslyn/issues/14600")]
+        public void GetSymbolInfo_01()
+        {
+            var source = @"
+class C
+{
+    static void Main()
+    {
+        bar x1 = (Alice: 1, ""hello"");
```

Comment:
It actually should work in the presence of errors. The error here was accidental, I think I will just make it more apparent.

---
In reply to: [97432815](https://github.com/dotnet/roslyn/pull/16659#discussion_r97432815) [](ancestors = 97432815)